content/webapp/components/Body/Body.tsx

```diff
@@ -166,12 +171,23 @@ const Body: FunctionComponent<Props> = ({
   comicPreviousNext,
   contentType,
 }: Props) => {
+  const { sliceMachine } = useToggles();
   const filteredBody = body
     .filter(slice => !(slice.type === 'picture' && slice.weight === 'featured'))
     // The standfirst is now put into the header
     // and used exclusively by articles / article series
     .filter(slice => slice.type !== 'standfirst');
 
+  const filteredUntransformedBody = untransformedBody
+    .filter(
+      slice =>
+        !(
+          slice.slice_type === 'editorialImage' &&
+          slice.slice_label === 'featured'
+        )
+    )
+    .filter(slice => slice.slice_type !== 'standfirst');
```

Comment:
Why are those two filtered?

Reply:
This is a good question and something I'm trying to understand myself at the moment. There is quite a bit of logic involved around pulling things out of the body and rendering them elsewhere under certain circumstances and I think there's room to improve a lot of this. But first step is understanding exactly what is happening and why.

The standfirst is straightforward in that we now render it separately in the header, so don't want it in the body. The featured stuff looks a bit more complicated. David created a [ticket about potential remodelling](https://github.com/wellcomecollection/wellcomecollection.org/issues/10753), I'll add notes to it when I'm confident what needs doing.